app/src/main/java/com/fs_sournary/weather/data/model/DataDaily.java

```diff
@@ -0,0 +1,455 @@
+    }
+
+    public void setMoonPhase(double moonPhase) {
+        mMoonPhase = moonPhase;
+    }
+
+    public double getPrecipIntensity() {
+        return mPrecipIntensity;
+    }
+
+    public void setPrecipIntensity(double precipIntensity) {
+        mPrecipIntensity = precipIntensity;
+    }
+
+    public double getPrecipIntensityMax() {
+        return mPrecipIntensityMax;
+    }
+
+    public void setPrecipIntensityMax(double precipIntensityMax) {
+        mPrecipIntensityMax = precipIntensityMax;
+    }
+
+    public int getPrecipIntensityMaxTime() {
+        return mPrecipIntensityMaxTime;
+    }
+
+    public void setPrecipIntensityMaxTime(int precipIntensityMaxTime) {
+        mPrecipIntensityMaxTime = precipIntensityMaxTime;
+    }
+
+    public double getPrecipProbability() {
+        return mPrecipProbability;
+    }
+
+    public void setPrecipProbability(double precipProbability) {
+        mPrecipProbability = precipProbability;
+    }
+
+    public String getPrecipType() {
+        return mPrecipType;
+    }
+
+    public void setPrecipType(String precipType) {
+        mPrecipType = precipType;
+    }
+
+    public double getTemperatureHigh() {
+        return mTemperatureHigh;
+    }
+
+    public void setTemperatureHigh(double temperatureHigh) {
+        mTemperatureHigh = temperatureHigh;
+    }
+
+    public int getTemperatureHighTime() {
+        return mTemperatureHighTime;
+    }
+
+    public void setTemperatureHighTime(int temperatureHighTime) {
+        mTemperatureHighTime = temperatureHighTime;
+    }
+
+    public double getTemperatureLow() {
+        return mTemperatureLow;
+    }
+
+    public void setTemperatureLow(double temperatureLow) {
+        mTemperatureLow = temperatureLow;
+    }
+
+    public int getTemperatureLowTime() {
+        return mTemperatureLowTime;
+    }
+
+    public void setTemperatureLowTime(int temperatureLowTime) {
+        mTemperatureLowTime = temperatureLowTime;
+    }
+
+    public double getApparentTemperatureHigh() {
+        return mApparentTemperatureHigh;
+    }
+
+    public void setApparentTemperatureHigh(double apparentTemperatureHigh) {
+        mApparentTemperatureHigh = apparentTemperatureHigh;
+    }
+
+    public int getApparentTemperatureHighTime() {
+        return mApparentTemperatureHighTime;
+    }
+
+    public void setApparentTemperatureHighTime(int apparentTemperatureHighTime) {
+        mApparentTemperatureHighTime = apparentTemperatureHighTime;
+    }
+
+    public double getApparentTemperatureLow() {
+        return mApparentTemperatureLow;
+    }
+
+    public void setApparentTemperatureLow(double apparentTemperatureLow) {
+        mApparentTemperatureLow = apparentTemperatureLow;
+    }
+
+    public int getApparentTemperatureLowTime() {
+        return mApparentTemperatureLowTime;
+    }
+
+    public void setApparentTemperatureLowTime(int apparentTemperatureLowTime) {
+        mApparentTemperatureLowTime = apparentTemperatureLowTime;
+    }
```

Comment:
ERROR: [PMD-CPD] 43 lines duplicated.
app/src/main/java/com/fs_sournary/weather/data/model/CurrentWeather.java

Reply:
Remove getter and setter method không dùng đến ạ ?